documentation/api-scenario/references/ErrorCodeReference.md

```diff
@@ -119,3 +119,36 @@ Example: The sku in request parameters is `default`, but actual return is `stand
 }
 
 ```
+
+## INCORRECT_LRO_FINAL_STATE_VIA
+
+This error is reported when the long running operation defined with 'final-state-via' in swagger but the initial request doesn't return a corresponding header described in the [x-ms-long-running-operation-options](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-long-running-operation).
+
+**How to fix**
+1. if a long running operation flagged with 'final-state-via': 'location or azure-async-operation or operation-location', the response of the inital request MUST include a corresponding header which contains the url for getting the final result of the operation.
+
+1. if a long running operation flagged with 'final-state-via': 'original-uri', then the operation should be a PUT/PATCH/DELETE operation of an ARM resource type and a GET operation should also be present under the corresponding swagger 'path'.
```

Comment:
INCORRECT_LRO_FINAL_STATE_VIA applies to both management plane and data plane, but this statement seems for ARM only